stackslib/src/net/tests/mod.rs

```diff
@@ -811,7 +890,10 @@ fn test_boot_nakamoto_peer() {
         NakamotoBootTenure::NoSortition(vec![NakamotoBootStep::Block(vec![next_stx_transfer()])]),
         // prepare phase for 2
         NakamotoBootTenure::NoSortition(vec![NakamotoBootStep::Block(vec![next_stx_transfer()])]),
-        NakamotoBootTenure::Sortition(vec![NakamotoBootStep::Block(vec![next_stx_transfer()])]),
+        NakamotoBootTenure::Sortition(vec![
```

Comment:
I'm curious why this was necessary?  It's not a problem; just making sure that having two transactions here isn't masking some underlying issue that arises when there's only one transaction.

Reply:
> Because of the way the Sortition/NoSortition tenures are structured in this test, there’s only one tenure during the prepare phase (aside from the 0 block which already needs the aggregate key to have been approved).

I think this has less to do with the boot plan and more to do with the PoX parameters passed into `.with_pox_constants()`.

Reply:
Yes -- it's both. If the prepare phase were longer, then there would be more sortitions in the prepare phase. But it's the combination of prepare phase length == 3 and the `NoSortition, NoSortition, Sortition` sequence that requires this behavior.